modules/custom/az_migration/migrations/az_site_frontpage.yml

```
@@ -0,0 +1,52 @@
+id: az_site_frontpage
+label: Site Front Page
+migration_group: az_migration
+migration_tags:
+  - Quickstart Settings Migration
```

Are we already using this tag on any of our other migrations?  Personally I think it'd be OK to use `Quickstart Content Migration` on this one, too...
```suggestion
  - Quickstart Content Migration
```

---

We aren't